vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/order_handler.rb

```diff
@@ -15,10 +15,8 @@ def initialize(occupancy)
       end
 
       def process
-        return unless occupancy.account_id?
-
         find_or_create_order
-        fulfill_order if occupancy.complete?
+        occupancy.attempt_order_complete!
```

Comment:
Part of me feels like what happens inside this method is more the responsibility of the order handler rather than the occupancy. I could be convinced otherwise, though, perhaps that it then has to know too much about the innards of occupancies. But on the other hand, as it is now, occupancy might know too much about orders. Thoughts?